hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/replication/GlobalHiveSyncTool.java

```diff
@@ -36,8 +38,12 @@
 
   private static final Logger LOG = LogManager.getLogger(HiveSyncTool.class);
 
+  private final HoodieHiveClient hoodieHiveClient;
+
   public GlobalHiveSyncTool(GlobalHiveSyncConfig cfg, HiveConf configuration, FileSystem fs) {
     super(cfg, configuration, fs);
+    ValidationUtils.checkArgument((syncClient instanceof HoodieHiveClient), "GlobalHiveSyncTool only support Hive Sync. For Glue meta sync, use HiveSyncTool with the same config");
```

Comment:
lets  not leak context of other sync tools into this one? remove "glue" mentions?